csrankings-a.csv

```diff
@@ -1067,7 +1067,7 @@ Amit P. Sheth,University of South Carolina,https://www.linkedin.com/in/amitsheth
 Amit Ranjan Trivedi,University of Illinois at Chicago,https://aeon.lab.uic.edu/Faculty.html,wlzmWLsAAAAJ
 Amit Sahai,Univ. of California - Los Angeles,http://www.cs.ucla.edu/~sahai,gqB23VMAAAAJ
 Amit Sengupta,DAIICT,http://www.daiict.ac.in/daiict/people/faculty.html,NOSCHOLARPAGE
-Amit Z. Dvir,Ariel University,https://www.ariel.ac.il/wp/amitd,S6wdYwIAAAAJ
+Amit Z. Dvir,Ariel University,https://www.ariel.ac.il/wp/amitd/, fYOS6GcAAAAJ

```

Comment:
Please remove space after comma.

Reply:
Hi, 

Thanks a lot. 

I moved the space after the comma